plugin/auto/README.md

```diff
@@ -15,9 +15,8 @@ zonefile. New or changed zones are automatically picked up from disk.
 
 ~~~
 auto [ZONES...] {
-    directory DIR [REGEXP ORIGIN_TEMPLATE [TIMEOUT]]
+    directory DIR [REGEXP [ORIGIN_TEMPLATE]]
```

Comment:
you've added a `[` here that wasn't here before.

Reply:
should it be here?

Reply:
Actually I was trying to make it stay consistent with the code, where `TEMPLATE` is optional. 

``` golang
			case "directory": // directory DIR [REGEXP [TEMPLATE]]
				if !c.NextArg() {
					return a, c.ArgErr()
				}
				a.loader.directory = c.Val()
                                 // ...

				// regexp
				if c.NextArg() {
                                         // ...
			        }
				// template is optional here
				if c.NextArg() {
					a.loader.template = rewriteToExpand(c.Val())
				}
```

I'm not sure whether you were making `TEMPLATE` optional on purpose. I think it is fine, but we should set a default value (e.g. `{0}`). WDYT?
BTW, I personally prefer a `return fast` coding style (return as soon as no args can be parsed).

Reply:
As far as my understanding of intent goes:
The optionality and the default is for both the regex and the template as they are quite correlated. That's why it wasn't separately shown as optional, but one "bundle".

`directory DIR [REGEXP ORIGIN_TEMPLATE]`

The code should reflect the default for this bundle to be `db\.(.*) {1}` afaik.

Reply:
That make sense. So we should raise an error when `REGEXP` is set but `ORIGIN_TEMPLATE` is not?